napari/utils/io.py

```diff
@@ -5,6 +5,7 @@
 from glob import glob
 from pathlib import Path
 from typing import Union, List, Optional, Tuple
```

Comment:
```suggestion
from typing import Union, List, Optional, Tuple, Generator
```
(If you choose to use the type hint below)